virttest/utils_backup.py

```diff
@@ -55,6 +55,19 @@ def __str__(self):
         return "Backup TLS failure: %s" % self.error_info
 
 
+class BackupCanceledError(BackupError):
+
+    """
+    Class of backup canceled exception
+    """
+
+    def __init__(self):
```

Comment:
whether consider add parameter e.g message to allow customized exception message

Reply:
It's NOT always an error when backup canceled, for examlpe, we can use 'domjobabort' to cancel the backup. So there is no error message produced by libvirt or kernel that need to be captured. And other user message is not worth to be put here, but should be put in somewhere such as logging.debug.